pepper-apis/src/main/resources/db-changes/schema/DDP-7004-auth0-logging-error-fix.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+
+    <changeSet author="evgeniipr" id="20210928-auth0-logging-error-fix">
+        <sql>
+insert into auth0_log_event_code (code, title, description)
+values
+('fapi', 'Operation on API failed', ''),
```

Comment:
Guess question is: is there a way that we can ensure that this list is up to date or that at least we are alerted that that we need to support some new code?
Better question: does the referential integrity here with all the possible codes helps us at all? Would it make sense to just store the codes as string? (this is really a question. My concern is that we keep having to worry about maintaining this list of codes up to date)

Reply:
I suggest to detalize an error text which will be written to log (and to Slack channel): it this text it's possible to show a small instruction what to do to fix the problem.

Reply:
In case of new code added by auth0 and this code contained by a log event: a detailed error text is logged.
It will look like:
```
Auth0LogEventRoute - Error: trying to insert null-value to column 'auth0_log_event_code_id' in table `auth0_log_event` . It's null because a log event code is not found in table `auth0_log_event_code` - the event contains an unknown code. Please, check URL `https://auth0.com/docs/monitor-auth0/logs/log-event-type-codes` for new codes and create Liquibase script with insert statements to insert new codes to table `auth0_log_event_code`.
Cause error:
...
```

Btw, I remember the discussion (may be with Jufeng) about a separate table for auth0 log event codes. It is decided to store codes in a separate table because it stores a detailed explanation of a code. It's too resource-expensive to save code details to every log event record.